include/net/net_linkaddr.h

```diff
@@ -45,6 +45,7 @@ enum net_link_type {
 	NET_LINK_BLUETOOTH,
 	NET_LINK_ETHERNET,
 	NET_LINK_DUMMY,
+	NET_LINK_CANBUS,
```

Comment:
I thought NET_LINK_DUMMY was supposed to be last.

Reply:
Should not matter, number is just a number.